t/proto_template/.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: 'Capture proposed version'
         id: 'capture'
         run: |
-          echo "PROPOSED_PROTO_VERSION=$(yq eval '.proto_version' version.yaml)" >> $GITHUB_OUTPUT
+          echo "PROPOSED_PROTO_VERSION=$(yq eval '.proto_version' version.yaml)" >> ${GITHUB_OUTPUT}
```

Comment:
nit: I still think yaml is useless here, it should just be a text file and you can cat the entire file instead of relying on yq...